commons-test/src/main/java/org/infinispan/commons/test/TestNGTestListener.java

```
@@ -20,6 +20,7 @@
    private final TestSuiteProgress progressLogger;
 
    public TestNGTestListener() {
+      EnvironmentCheck.checkJVMVersion();
```

For TestNG modules we can throw an exception in `TestNGSuiteChecksTest.beforeSuite()` instead of calling `System.exit()`

---

ok

---

For JUnit I've moved the check to the testRunStarted method